<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>860</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>860</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1100</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>860</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1100</width>
     <height>860</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1100</width>
     <height>860</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0.058, y1:0.0685288, x2:0.997304, y2:0.958, stop:0 rgba(255, 170, 127, 255), stop:0.976109 rgba(85, 170, 255, 255))
}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>KAYIT OL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="kayitOl">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>650</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(143, 220, 255);
border-radius:20px;</string>
    </property>
    <property name="text">
     <string>Kayıt Ol</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ileriButon">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Custom-Icon-Design-Mono-General-1-Forward.ico</normaloff>Custom-Icon-Design-Mono-General-1-Forward.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="geriButon">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(232, 232, 232);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Custom-Icon-Design-Mono-General-1-Back.ico</normaloff>Custom-Icon-Design-Mono-General-1-Back.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>E-Posta:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="epostaLine">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Şifre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sifreLine">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Şifre Doğrulama:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sifreLine2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="sifreGoster">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>620</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Şifreyi Göster</string>
    </property>
   </widget>
   <widget class="QLabel" name="hata">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>730</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 4);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>epostaLine</tabstop>
  <tabstop>sifreLine</tabstop>
  <tabstop>sifreLine2</tabstop>
  <tabstop>sifreGoster</tabstop>
  <tabstop>kayitOl</tabstop>
  <tabstop>geriButon</tabstop>
  <tabstop>ileriButon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
